Frontend/android-native/mobility-common/src/main/java/in/juspay/mobility/common/MobilityCommonBridge.java

```diff
@@ -2328,6 +2313,11 @@ public static class ZoomType {
         public static final String ZOOM = "ZOOM";
     }
 
+    public static class MarkerType {
+        public static final String SPECIAL_ZONE_MARKER = "SPECIAL_ZONE_MARKER";
```

Comment:
@ashishsingh101 can we create an enum here instead of string.

Reply:
sure